packages/flutter/lib/src/material/icon.dart

```diff
@@ -65,13 +83,13 @@ class Icon extends StatelessComponent {
       category = parts[0];
       subtype = parts[1];
     }
-    // TODO(eseidel): This clearly isn't correct.  Not sure what would be.
-    // Should we use the ios images on ios?
-    String density = 'drawable-xxhdpi';
+    // TODO(kgiesing): Should selector logic be moved down a level so that
+    // any generic asset can be resolved using consistent contextual logic?
```

Comment:
Right, I guess the question is whether the third layer (predefined set of breakpoints) is in material or underneath it.  My vote would be underneath since I think this has more to do with Flutter application development than material design.